Nesteo.Server.DataImport/Program.cs

```diff
@@ -0,0 +1,420 @@
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Gartenbaumläufer" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Gartenrotschwanz" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Grauschnäpper" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Haussperling" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Hohltaube" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Kleiber" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Kohlmeise" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Kohlmeise x Blaumeise" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Meise" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Meise unbestimmt" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Rotkehlchen" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Star" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Sumpfmeise" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Tannenmeise" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Trauerschnäpper" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "Zaunkönig" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "unbestimmt" });
+//            dbContext.Species.Add(new SpeciesEntity { Name = "unbekannt" });
+
+        }
+
+        private static async Task GenerateNestingBoxes(NesteoDbContext dbContext, Stammdaten record, UserEntity user)
+        {
+            OwnerEntity ownerEntity;
+            RegionEntity regionEntity;
+            Material material;
+            HoleSize holeSize;
+
+            // Search for ownerId
+            if (record.Eigentumer != " ")
+            {
+                ownerEntity = (from o in dbContext.Owners where o.Name == record.Eigentumer select o).FirstOrDefault();
+            }
+            else
+            {
+                ownerEntity = null;
+            }
+
+            // Search for regionId
+            if (record.Ort != " ")
+            {
+                regionEntity = (from r in dbContext.Regions where r.Name == record.Ort select r).FirstOrDefault();
+            }
+            else
+            {
+                regionEntity = null;
+            }
+
+            // Get Material Value
+            if (record.Material.StartsWith("Holbeton"))
+            {
+                material = Material.WoodConcrete;
+            }
+            else if (record.Material == "Holz unbeschicht")
+            {
+                material = Material.UntreatedWood;
+            }
+            else if (record.Material == "Holz beschicht")
+            {
+                material = Material.TreatedWood;
+            }
+            else
+            {
+                material = Material.Other;
```

Comment:
Maybe print these out so we can make sure that nothing get's wrongly detected as `unknown`?